doc/dev/index.rst

```diff
@@ -530,38 +598,106 @@ is deemed to be an integration test. Integration tests are organized into
+  run RBD tests using actual Ceph clusters, with and without qemu
+
+`rgw <https://github.com/ceph/ceph-qa-suite/tree/master/suites/rgw>`_
+  run RGW tests using actual Ceph clusters
+
+`smoke <https://github.com/ceph/ceph-qa-suite/tree/master/suites/smoke>`_
+  run tests that exercise the Ceph API with an actual Ceph cluster
+
+`teuthology <https://github.com/ceph/ceph-qa-suite/tree/master/suites/teuthology>`_
+  verify that teuthology can run integration tests, with and without OpenStack
+
+`upgrade <https://github.com/ceph/ceph-qa-suite/tree/master/suites/upgrade>`_
+  for various versions of Ceph, verify that upgrades can happen
+  without disrupting an ongoing workload
+
+.. _`ceph-deploy man page`: ../../man/8/ceph-deploy
+.. _`ceph-disk man page`: ../../man/8/ceph-disk
+
+teuthology-describe-tests
+-------------------------
+
+In February 2016, a new feature called ``teuthology-describe-tests`` was
+added to the `teuthology framework`_ to facilitate documentation and better
+understanding of integration tests (`feature announcement
+<http://article.gmane.org/gmane.comp.file-systems.ceph.devel/29287>`_).
+
+The upshot is that tests can be documented by embedding ``meta:``
+annotations in the yaml files used to define the tests. The results can be
+seen in the `ceph-qa-suite wiki
+<http://tracker.ceph.com/projects/ceph-qa-suite/wiki/>`_.
 
 Running integration tests
 -------------------------
 
-Some Ceph developers have access to the `Sepia lab`_ and are allowed to
-schedule integration tests there. The developer nick shows in the test
-results URL and in the first column of the Pulpito dashboard.
+Given that - as a new Ceph developer - you will typically not have access
+to the `Sepia lab`_, you may rightly ask how you can run the integration
+tests in your own environment.
 
-"How can I run the integration tests in my own environment?" you may ask.
 One option is to set up a teuthology cluster on bare metal. Though this is
 a non-trivial task, it `is` possible. Here are `some notes
 <http://docs.ceph.com/teuthology/docs/LAB_SETUP.html>`_ to get you started
 if you decide to go this route.
 
-If you have access to an OpenStack tenant, you have another option:
-teuthology has an OpenStack backend, which is documented `here
+If you have access to an OpenStack tenant, you have another option: the
+`teuthology framework`_ has an OpenStack backend, which is documented `here
 <https://github.com/dachary/teuthology/tree/openstack#openstack-backend>`_.
 This OpenStack backend can build packages from a given git commit or
-branch, provision VMs in a public or private OpenStack
-instance (sometimes referred to as a "cloud"), install the packages and run
-integration tests on those VMs. This process is controlled using a tool
-called `ceph-workbench ceph-qa-suite`_. This tool also automates publishing
-of test results at http://teuthology-logs.public.ceph.com. 
+branch, provision VMs in a public or private OpenStack instance (sometimes
```

Comment:
I would remove **in a public or private OpenStack instance (sometimes referred to as a "cloud")**